### ISSUE TITLE
site examples not in sync with code release

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/caliper#291]
```
What steps will reproduce the problem?
1. caliper-1.0-beta-1-all does not contain com.google.caliper.BeforeExperiment;

2. example code on the website references this annotation for example:

https://code.google.com/p/caliper/source/browse/examples/src/main/java/examples/
CopyArrayBenchmark.java


```

Original issue reported on code.google.com by `pbir...@gmail.com` on 4 Apr 2014 at 12:56

**Comment by blundell** [comment]
Commenting so I get updates :-)

**Comment by netdpb** [comment]
I think this issue was fixed with 1.0-beta-2.
